mkdocs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This is a small shell script that...

- Copies the README to docs/index.md
- Copies the LICENSE to docs/license.md
- Fixes references/links in the copied docs/index.md
- Generates the static website in `site`